permission_handler_platform_interface/lib/src/permissions.dart

```diff
@@ -95,42 +96,46 @@ class Permission {
   static const speech = Permission._(14);
 
   /// Android:
-  ///   On Android < TIRAMISU the `READ_EXTERNAL_STORAGE` and
-  ///   `WRITE_EXTERNAL_STORAGE` permissions are requested (depending on the
-  ///   definitions in the AndroidManifest.xml) file.
   ///   On Android TIRAMISU and higher this permission is deprecrated and
-  ///   always returns `PermissionStatus.denied`, instead use
-  ///   `Permission.photos`, `Permission.video`, `Permission.audio` or
-  ///   `Permission.manageExternalStorage`. For more information see our [FAQ](https://pub.dev/packages/permission_handler#faq).
-  /// iOS: Access to folders like `Documents` or `Downloads`. Implicitly
-  /// granted.
+  ///     always returns `PermissionStatus.denied`, instead use
+  ///     `Permission.photos`, `Permission.video`, `Permission.audio` or
+  ///     `Permission.manageExternalStorage`. For more information see our
+  ///     [FAQ](https://pub.dev/packages/permission_handler#faq).
+  ///   On Android < TIRAMISU the `READ_EXTERNAL_STORAGE` and
+  ///     `WRITE_EXTERNAL_STORAGE` permissions are requested (depending on the
+  ///     definitions in the AndroidManifest.xml) file.
+  /// iOS: Access to folders like `Documents` or `Downloads`. Implicitly granted.
   static const storage = Permission._(15);
 
   /// Android: Ignore Battery Optimizations
+  /// iOS: Nothing
   static const ignoreBatteryOptimizations = Permission._(16);
 
   /// Android: Notification
   /// iOS: Notification
   static const notification = Permission._(17);
 
   /// Android: Allows an application to access any geographic locations
-  /// persisted in the user's shared collection.
+  ///   persisted in the user's shared collection.
+  /// iOS: Nothing
   static const accessMediaLocation = Permission._(18);
 
-  /// When running on Android Q and above: Activity Recognition
-  /// When running on Android < Q: Nothing
+  /// Android:
+  ///   When running on Android Q and above: Activity Recognition
+  ///   When running on Android < Q: Nothing
   /// iOS: Nothing
   static const activityRecognition = Permission._(19);
 
   /// The unknown only used for return type, never requested
   static const unknown = Permission._(20);
 
+  /// Android: Nothing
   /// iOS 13 and above: The authorization state of Core Bluetooth manager.
   /// When running < iOS 13 or Android this is always allowed.
-  static const bluetooth = Permission._(21);
+  static const bluetooth = PermissionWithService._(21);
 
   /// Android: Allows an application a broad access to external storage in
-  /// scoped storage.
+  ///   scoped storage.
```

Comment:
nit: redundant spacing

```suggestion
  /// scoped storage.
```